src/server/language.py

```diff
@@ -52,6 +52,7 @@ class Language(object):
     NL = _Language('Nederlands', 'nl')
     PT = _Language('Português', 'pt')
     RU = _Language('Русский', 'ru')
+    UK = _Language('Українська', 'uk')
```

Comment:
Is this correct @tymosh ? I Googled it so apologies if wrong!

Reply:
Yes, this is absolutely right